src/runtime_src/core/pcie/linux/system_linux.cpp

```diff
@@ -85,6 +99,18 @@ glibc_info()
   return _pt;
 }
 
+static std::string machine_info()
+{
+  static const char node[] = MACHINE_NODE_PATH;
```

Comment:
What's the point of node here?  Why not use MACHINE_NODE_PATH directly line 106?

Reply:
Removed my brain dead artifact.

Reply:
Yes, that will work as well.  

Personally for me, I like to use a static const variable to contain this type of information especially if the value is determine via the compiler environment.  Having this in a static const variable also enables one to easily inspect this value via the debugger, if needed in the future.  

Again, not a big deal.  It is just a coding pattern I like to do especially if it is being used as an argument into a function/method that is difficult to step into.

Reply:
Sounds like you suggest working around the misuse of #define :-)  
In reality all the #defines should be `constexpr char blah[] = "blahblah"`   
Should we outlaw #defines in C++ code?